packages/lib/models/settings/builtInMetadata.ts

```diff
@@ -1571,6 +1571,10 @@ const builtInMetadata = (Setting: typeof SettingType) => {
 			type: SettingItemType.Bool,
 			public: true,
 			storage: SettingStorage.File,
+			label: () => _('Use beta encryption'),
+			description: () => _('Set beta encryption methods as the default methods. This applies to all clients and takes effect after restarting the app'),
```

Comment:
Thanks for adding this but please don't make it translatable since it's temporary, and that way we reduce the burden on the translator

Reply:
I see. I've removed the `_()` here.